### ISSUE TITLE
Change federal agency dropdown to view only style on organization name and mailing address page

### ISSUE THREAD

**Issue by gabo0oo** [cisagov/manage.get.gov#2217]
### Issue description

In the past, domain managers were able to edit the federal agency dropdown that appears on the organization name and mailing address page. We've now added a federal agency table in /admin that populates this dropdown, so we don't want domain managers to change this field. As a quick fix, this dropdown was disabled, but for better UX and accessibility, we should be using a different style for presenting this info. 

### Acceptance criteria

- [ ] Change federal agency field to view only version ([see Figma](https://www.figma.com/design/xFtGLHVrhp0lvwh0gYWnbx/.gov-Designs-CISA?node-id=6736-75843&t=GxvordiRcvO0hCTL-11)) 

### Links to other issues

🔄 Relates to #1793 
For past discussions on this and accessibility, see [this slack thread.](https://cisa-corp.slack.com/archives/C05BGB4L5NF/p1716490664872229?thread_ts=1715972191.641599&cid=C05BGB4L5NF)

### Prioritization Comments
**WCAG Conformance Level:** 2.1 AA
**WCAG Standards that apply:** 
1.3.1 - Info and Relationships (Critical)
2.4.4 - Link Purpose(Non-critical)
1.4.1 - Use of Color (Critical)
4.1.2 - Name, Role, Value (Critical)
**Critical Failure if Not Met?:**  Yes
**Prioritization:** High/Must Do
**Reason:** This issue affects accessibility and usability due to the disabled dropdown, which may not be clear to users, especially those using assistive technologies. Shifting the dropdown to a view-only field will improve clarity and align with WCAG standards, ensuring that information is accessible and clearly presented.

**Comment by Katherine-Osos** [comment]
@gabydisarli Was this already fixed (or soon to be fixed) as part of the org model work?

**Comment by gabydisarli** [comment]
@Katherine-Osos I've updated the ticket, should be good to go!

**Comment by gabydisarli** [comment]
@katypies @abroddrick @Katherine-Osos I updated the link to the Figma (I think that's maybe why refinement was on here?). Anything else I can provide?

**Comment by Katherine-MN** [comment]
Removed "skipped design" to reflect that design documentation has already been updated to reflect these changes.
